### PR TITLE
Let only complete works count towards the quota

### PR DESCRIPTION
**Pull request by jengelh**
Ensure that the 20% quota can not be fulfilled by releasing only
incomplete, partial works.

### CODE REVIEW

**Comment by willbarton** [comment]
How would that affect works-in-progress? If an agency develops in the open by default, does code-in-progress count? Should it?

I understand the intent (and I think this is a larger problem with setting an arbitrary quota), but I'd rather see more agencies developing in the open by default (like 18F and CFPB do), and I think this policy should encourage that. While completed, end-of-project code dumps would be preferable to no code at all, development in the open from the beginning is the ideal, in my opinion. I'm afraid of how decision-markers at agencies could interpret "complete work" in this context.

**Comment by jengelh** [comment]
So let me rephrase/elaborate.

The intent was to discourage code dumps (usually as compressed archive files like .zip/.tar/etc, or an upload to something like Github) which are incomplete to the point that the program executable, assuming all third-party dependencies (system libraries) are satisfied, cannot be run because of willfull exclusion of files essential to its operation.

Perhaps this could be worded as such:

Works can count towards the quota as soon as the code, as measured by the publicly available copy, is runnable to the point that it would be of use to the agency itself.

**Comment by chipmonkey** [comment]
I would rather see a bad code dump than nothing at all, although I do see your point, @jengelh, so I'm not sure how to balance that.  "Runnable" may limit things like modules and libraries which could be solid OSS contributions on their own, but not really executable.  I personally feel that a stronger library of modules that each fulfill common needs may be a better starting point for OSS reuse than compelte monoliths which are unlikely to fill all of a new project's requirements.  This seems like it may conflict with that a bit.

Actually, if you just take out "runnable" and leave just "to the point that it would be of use" or, perhaps "can be re-used by others" in your example?

**Comment by mattbailey0** [comment]
Thank you for your input - I'm closing this pull request given that the final language has now been released. Please feel free to reopen or create a new issue or pull if you'd like to continue the conversation!
